### ISSUE TITLE
«How to contribute» in Readme

### ISSUE THREAD

**Issue by signalwerk** [johnboiles/obs-mac-virtualcam#39]
Maybe you could add a section in the Readme to try to involve more people. There are 400+ stars maybe there are people searching for opportunities?

A suggestion for a start (copy & past ready for MD):

```md
## How to contribute
* **Developer** · Build the plugin from source and fix a problem from the «Known Issues» in the readme.
* **Developer**· Build the plugin from source and go to the [issues](https://github.com/johnboiles/obs-mac-virtualcam/issues) and fix something
* **Developer**· Build the plugin from source and do a Pull Request for a CI. For example for [Travis CI](https://travis-ci.org/). The built plugin should end up in the Github-Release of this Repo. 
* **Tec People** · Suggest a readme/file with «Common Problems». People who try the plugin have sometimes problems with for example permission rights. See [issue #35](https://github.com/johnboiles/obs-mac-virtualcam/issues/35). Keep an eye on the issues and add sections if necessary.
* **Tec People** · Watch this repo and whenever people come with questions in the [issues](https://github.com/johnboiles/obs-mac-virtualcam/issues) try to be the first to answer it. Even if you are not a developer you can help people by pointing in the right direction. It helps the developer to focus on writing code.
```

**Comment by gxalpha** [comment]
Great idea!

Added to #11, though not to readme as I think it's getting quite long

**Comment by signalwerk** [comment]
@gxalpha thank you for doing it as a PR! I also like the idea to have it in a separate doc. I think it would be good to have in the Readme a Link to that file.

Now the commit it's tight to your merge request of #11 . This is not ideal because it has nothing to do with each other. Can you do a revert on your master? I made a separate branch just for that https://github.com/johnboiles/obs-mac-virtualcam/pull/40

Sorry for the mess... I should have done that from the beginning.

**Comment by johnboiles** [comment]
I merged #40. Thanks y'all!!
